multimodal/tests/unittests/predictor/test_predictor.py

```diff
@@ -677,6 +678,31 @@ def test_load_ckpt():
     npt.assert_equal(predictions_prob, predictions2_prob)
 
 
+def test_fttransformer_load_ckpt():
+    download("https://automl-mm-bench.s3.amazonaws.com/ft_transformer_pretrained_ckpt/iter_2k.ckpt", "./")
```

Comment:
There is no documentation for how to download a pretrained ft transformer checkpoint. If the benchmark results can show the pretrained is better than the random initialized, we can use the pretrained as the default. That is, we download checkpoint and initialize ft transformer internally. It would make it easier for users.

Reply:
If you'd like to run a benchmark of pre-trained vs random, we can run it on AutoML Benchmark after the code is in a state where I can specify the pre-trained weights to load via the model hyperparameters through TabularPredictor.

Reply:
Can we remove this line since we already support downloading it internally?